easybuild/easyconfigs/p/Python/Python-2.7.18-GCCcore-10.2.0.eb

```diff
@@ -43,9 +43,9 @@ exts_list = [
         'use_pip': False,
         'checksums': ['c67aa55db532a0dadc4d2e20ba9961cbd3ccc84d544e9029699822542b5a476b'],
     }),
-    ('pip', '20.2.3', {
+    ('pip', '20.1.1', {
```

Comment:
@algiar Please use `20.3.4` instead (the latest version of `pip` which still supports Python 2), see discussion in https://github.com/easybuilders/easybuild-easyconfigs/issues/12016#issuecomment-787052638.